huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go

```diff
@@ -403,11 +439,56 @@ func resourceGaussRedisInstanceV3Create(ctx context.Context, d *schema.ResourceD
 		PollInterval: 20 * time.Second,
 	}
 
-	_, err = stateConf.WaitForStateContext(ctx)
+	res, err := stateConf.WaitForStateContext(ctx)
 	if err != nil {
 		return diag.Errorf("error waiting for instance (%s) to become ready: %s", instance.Id, err)
 	}
 
+	if publicIps := d.Get("public_ips").(*schema.Set).List(); len(publicIps) > 0 {
```

Comment:
Seems this is too complex, how about we just adding a new binding eip resource to associate eip to redis node, that will be more convenient.